server/routes/api/choropleth.py

```diff
@@ -241,7 +241,7 @@ def geojson():
           "current_geo": place_dcid
       }
   }
-  return Response(json.dumps(result), 200, mimetype='application/json')
+  return lib_util.gzip_compress_response(result, is_json=True)
```

Comment:
maybe we should also use gzip for entity-geojson?

Reply:
it didn't seem as necessary yet. it would be more effective to add caching, but it's now a POST. we can do that as a follow up